### ISSUE TITLE
Add tags to the project

### ISSUE THREAD

**Issue by freephile** [freephile/meza#5]
The project is run by tags and releases. There have been no tags or releases since the project was cloned to the NASA organization. So, create a tag "35.0.0" for the GRC-ATF branch, then create a new tag 39.0.0 for the REL1_39 branch and create a release for the merged product.

We want tags on the development branch because `meza --version` looks for tags with `sudo git --git-dir=/opt/meza/.git describe --tags` which is currently failing because there are zero tags. We can prevent the failure by using the `--always` option (that would return the latest git hash) but we really need to start using tags and releases.

**Comment by freephile** [comment]
`git describe --tags` returns
[closest-tag]-[num_commits-between_tag_and_head]-[sha1_short]

For my (local workstation) clone of my fork of the NASA repo (so my origin is freephile/meza), I added the enterprisemediawiki repo as a third remote, and fetched all the tags. Then I pushed the tags into my repo.

```
git remote add emw https://github.com/enterprisemediawiki/meza.git
git remote add nasa https://github.com/nasa/meza.git
git fetch emw --tags
git push origin --tags
```

Then I could `sudo git fetch origin --tags` in my VirtualBox environments, and `describe` works now.

```
[userx@localhost meza]$ sudo git --git-dir=/opt/meza/.git describe --tags
31.12.0-368-gcd9359f
```

But, the canonical (NASA) repo still needs **new** tags to represent the 35.x version as well as the pending 39.x version.

**Comment by freephile** [comment]
### Existing tag history

Here is partial output of the tag history (sorted)
`git tag -l "31.*" | sort -V`


```
31.5.0
31.6.0
31.7.0
31.8.0
31.8.0.1
31.8.1
31.8.2
31.8.3
31.8.4
31.9.0
31.10.0
31.10.1
31.11.0
31.12.0
31.13.0
31.14.0
31.15.0
31.16.0
31.17.0
31.18.0
```

And then, the details of the last tag

```
[userx@localhost meza]$ git show 31.18.0
commit ebfff7097be7d593afc9e59dce410e7ab6664e09 (tag: 31.18.0, origin/31.x)
Merge: c0a91e7 9ca5321
Author: James Montalvo <edwin.j.montalvo@nasa.gov>
Date:   Mon Nov 7 14:44:01 2022 -0600

    Merge pull request #1307 from enterprisemediawiki/remove-mod-proxy-html
    
    fix: remove apache mod_proxy_html which was unused and isn't available on RHEL
```

**Comment by freephile** [comment]
### Solved
You can now see the long list of tags I created for the 35.x series as well as a new tag for the beginning of the REL1_39 branch and milestone.

[userx@localhost meza]$  `git log --oneline --decorate --tags --no-walk`

I pushed these tags up into this repo.

```
58bc5b6 (tag: 39.0.0) Major upgrade of Meza for MediaWiki 1.39
76052b2 (tag: 35.23.0, origin/35.x) fix: handle directory change in elastic-rebuild-all.sh
a47e1c5 (tag: 35.22.0) Merge pull request #37 from nasa/35.x
5598ea0 (tag: 35.21.0) Merge pull request #30 from nasa/35.x
444b057 (tag: 35.20.0) fix: add --doshared to update.php to fix MW 1.31-->1.35 upgrade
487c41b (tag: 35.19.0) Merge pull request #20 from nasa/35.x
fabcf0c (tag: 35.18.0, origin/main, origin/HEAD, main) Merge pull request #19 from nasa/35.x
764ba9f (tag: 35.17.0) Adjust packages for MW 1.35/RL8
9cdc90d (tag: 35.16.0) Merge pull request #15 from nasa/35.x
1239d06 (tag: 35.15.0) Merge pull request #14 from nasa/35.x
8553dbd (tag: 35.14.0) Merge pull request #12 from nasa/35.x
bbbec24 (tag: 35.13.0) Merge pull request #10 from nasa/35.x
7dce0fc (tag: 35.12.0) bump header tabs extension to 2.2.1 to get support for 1.35
d0b8015 (tag: 35.11.0) Merge pull request #9 from nasa/35.x
763b683 (tag: 35.10.0) Merge branch '35.x' of https://github.com/nasa/meza into 35.x
ed39f11 (tag: 35.9.0) Merge branch '35.x' of https://github.com/enterprisemediawiki/meza into 35.x
96aebe1 (tag: 35.8.0) Merge pull request #8 from nasa/35.x
5adc699 (tag: 35.5.0) 35.x (#6)
ed4e93f (tag: 35.4.0) feat: support memcached/saml on 35.x/RHEL8
9bf5f30 (tag: 35.3.0) Merge pull request #1309 from djflux/rhel8-35.x
ebfff70 (tag: 31.18.0, origin/31.x) Merge pull request #1307 from enterprisemediawiki/remove-mod-proxy-html
5997d45 (tag: 35.2.0) Merge pull request #1306 from djflux/rhel8-35.x
d62927f (tag: 35.1.0) Merge branch 'rhel8-35.x' of github.com:djflux/meza into rhel8-35.x
c1f88dc (tag: 35.0.0) Merge branch 'rhel8-35.x' of github.com:djflux/meza into rhel8-35.x
85964d4 (tag: 31.17.0) Merge pull request #1301 from enterprisemediawiki/31.17.0-release
1780b6f (tag: 31.16.0) Merge pull request #1293 from enterprisemediawiki/31.16.0-release
707e8fa (tag: 31.15.0) Update RELEASE-NOTES.md (#1289)
cdcde61 (tag: 31.14.0) 31.14.0 release (#1285)
```

**Comment by freephile** [comment]
> create a new tag 39.0.0 for the REL1_39 branch and create a release for the merged product.

Creating an issue to track this work.

- [ ] Create annotated tag for 1.39
- [ ] Create Release Notes for 1.39
- [ ] Create a Release for 1.39
